x11-terms/cool-retro-term/cool-retro-term-1.2.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg
+
+QTW_PN="qmltermwidget"
+QTW_PV=63228027e1f97c24abb907550b22ee91836929c5
+QTW_P="${QTW_PN}-${QTW_PV}"
+
+DESCRIPTION="Terminal emulator with an old school look and feel"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term/"
+
+SRC_URI="
+	https://github.com/Swordfish90/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/Swordfish90/${QTW_PN}/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
+"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+RESTRICT="test"
+
+DEPEND="
+	dev-qt/qtdeclarative:5[localstorage]
+	dev-qt/qtgraphicaleffects:5
+	dev-qt/qtquickcontrols:5[widgets]
+	dev-qt/qtsql:5
+	dev-qt/qtwidgets:5
+"
```

Comment:
```
 * VDB: detected possibly incorrect RDEPEND (x11-terms/cool-retro-term-1.2.0)
 *                > dev-qt/qtcore:5
 *                > dev-qt/qtgui:5
 *                > dev-qt/qtnetwork:5
 * dev-qt/qtsql:5 < 
```
Please add the dependencies on the right hand side, and check if dev-qt/qtsql is really needed.

Reply:
You are right: SQL instructions used depends on `dev-qt/qtdeclarative:5[localstorage]` and not on `dev-qt/qtsql:5`. I have forgotten to add some QtQuick dependencies such as `dev-qt/qtgraphicaleffects:5`, `dev-qt/qtquickcontrols:5[widgets]` and `dev-qt/qtquickcontrols2:5[widgets]`.